.pipelines/submodules-pipeline.yaml

```diff
@@ -186,7 +186,7 @@ stages:
               arch: amd64
               name: npm
               os: windows
-              os_version: ltsc2019
+              os_version: "1809"
```

Comment:
let's remove all windows 2019 parts since Windows NPM is only supported on ws2022

Can remove this `npm_windows2019_amd64` section altogether